.travis.yml

```diff
@@ -32,3 +32,39 @@ deploy:
   skip_cleanup: true
   on:
     tags: true
+
+after_deploy:
+  - 'git clone https://github.com/Appliscale/rpmbuild.git'
```

Comment:
Maybe we can create a script, e.g. `release.sh`, and just call it here? With so many steps travis config is starting to looks unclear.